### PR TITLE
Fixes: mozilla/browserid#1859

### PR DESCRIPTION
**Pull request by david-caro**
Using the new version string format (and some minor improvements)

### CODE REVIEW

File: scripts/rpmbuild.sh [hunk 1]
```diff
@@ -3,14 +3,90 @@
...
+    || {
+        [[ -d locale ]] \
+        || svn co  http://svn.mozilla.org/projects/l10n-misc/trunk/browserid/locale
+    }   
+    LOCALE_REV="$(svn info locale/ | sed -n -e "s,^Rev.*n: ,,p")"
+}
+
+get_ver_string()
+{
+    branch="$(git branch | grep '*')"
```
Comment (gene1wood) [inline scripts/rpmbuild.sh]: Maybe something like this instead? : 

branch="$(git symbolic-ref HEAD 2>/dev/null)" || branch="(unnamed branch)"
branch=${branch##refs/heads/}

File: scripts/rpmbuild.sh [hunk 2]
```diff
@@ -3,14 +3,90 @@
...
+    branch="${branch:2}"
```
Comment (gene1wood) [inline scripts/rpmbuild.sh]: And remove this line

File: scripts/rpmbuild.sh (hunk 1 above)
Comment (david-caro) [inline scripts/rpmbuild.sh]: That's better thanks

**Comment by david-caro** [comment]
I'm having problems testing this new code, 'cause when building it complains about a missing file named  resources/static/common/js/lib/bidbundle.js, that existed before (in the tain-2012.07.30 branch at least).
When executing the env CONFIG_FILES=config/l10n-all.json scripts/compress part.

Somebody can help me get past this so I can fully test it?
Thaks!

**Comment by david-caro** [comment]
BTW. the file: https://github.com/mozilla/browserid/blob/train-2012.07.30/resources/static/common/js/lib/bidbundle.js

**Comment by seanmonstar** [comment]
@david-caro That file used to be a symlink, but that was removed to place nice on Windows. `npm install` will now make that file directly.

You can fix it doing:

```
rm -rf node_modules/
npm install
```

**Comment by lloyd** [comment]
this is out of date, we will be more responsive moving forward.  But given no action in 6 months and that it doesn't merge cleanly, I'm going to close it down.

**Pull request closed without merging**